scripts/blant-clusters.sh

```diff
@@ -212,31 +213,53 @@ for k in "${Ks[@]}"; do
 	sort -nr | # sort the above output by number of nodes in the near-clique
 	hawk 'BEGIN{ numCliques=0 } # post-process to remove duplicates
 	    {
-		delete S; seenColon=0;
+		delete S;
 		numNodes=$1
 		edgeHits=$2;
 		for(i=3;i<=NF;i++) ++S[$i]
 		ASSERT(length(S)==numNodes,"mismatch in numNodes and length(S)");
 		    for(i=1;i<=numCliques;i++) {
 			    same=0;
 			    for(u in S) if(u in cluster[i])++same;
-			    if(same == length(S)) break;
+			    if(same >= length(S)*'$t') break;
 		}
-		if(numCliques==0 || same < length(S)) {
+		if(numCliques==0 || same < length(S)*'$t') {
 			    maxEdges=choose(length(S),2);
-			    ++numCliques; printf "%d nodes, %d of %d edges from k '$k' (%g%%):",
-			length(S), edgeHits, maxEdges, 100*edgeHits/maxEdges
+			    ++numCliques; 
+				printf "%d %d '$k'",length(S),edgeHits
```

Comment:
I'm OK with shortening the output a bit, but the edge density I think needs to be printed.. plus please leave the colon before the list of nodes, I need that to be able to easily isolate the preamble from the list of nodes.

Reply:
Second output is different to first output. I shorten the first output because we don't need it formatted yet.  The first output is only used internally. The final output stays the same as in previous version.

Reply:
AHA! OK. Got it. Thanks.